build.gradle.kts

```diff
@@ -30,6 +30,7 @@ import org.gradle.api.tasks.testing.logging.TestLogEvent
 import org.jetbrains.dokka.gradle.DokkaTask
 import org.jetbrains.gradle.ext.JUnit
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.jetbrains.kotlin.gradle.plugin.KotlinCompilation
```

Comment:
> Note that IntelliJ IDE support for this feature is something different and not implemented yet.

Does that mean that the IDE will show an error but compilation will work? Or what else is missing in the IDE support?

Reply:
If you set `Run tests using` to `Gradle` in the IntelliJ IDEA settings then running functional tests that access `internal` objects will indeed work, but the IDE will underline access of such objects in red and give false hints.

Reply:
Maybe I miss something, but why do you activate this setting if it is not used anyway because of missing IDE support?

Reply:
Simply because I first thought it would work in the IDE this way, too. And as it was quite hard to find out the correct syntax for this, I wanted to keep the code even if it the feature is currently unused, so that we have this part already done when the IDE support will also be coming later.

Reply:
I think it's worth adding a code comment for this.

Reply:
I was hoping the commit message would suffice, but I've now also added a code comment.

Reply:
Thanks.